### PR TITLE
(EAI-294) Eval data generation for just LLMs

### PR DESCRIPTION
**Pull request by mongodben**
Jira: https://jira.mongodb.org/browse/EAI-294

## Changes

- Added eval framework data generator for generating conversation response just with an LLM
  - Helpful for assessing LLM's built-in knowledge/performance on a topic.
- Add implementations testing all our questions for GPT 3.5 and GPT 4 from Azure OpenAI service 
- Export our system prompt from `chatbot-server-mongodb-public` for use in LLM evaluation

## Notes

-

### CODE REVIEW

File: packages/mongodb-chatbot-evaluation/src/generate/generateLlmConversationData.test.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        sleepMs: SLEEP_TIME,
+      });
+    const start = Date.now();
+    await generateLlmConversationDataWithSleep({
+      runId: new ObjectId(),
+      testCases,
+    });
+    const end = Date.now();
+    expect(end - start).toBeGreaterThan(SLEEP_TIME * numSleeps);
+    // and confirm that it's not just taking a long time to run (which it shouldn't given the mocks)
```
Comment (nlarew) [inline packages/mongodb-chatbot-evaluation/src/generate/generateLlmConversationData.test.ts:65]: Were you running into an issue with this? Curious as to why we include this case.

File: packages/mongodb-chatbot-evaluation/src/generate/generateLlmConversationData.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      await sleep(sleepMs);
+    }
+
+    return { generatedData, failedCases };
+  };
+};
+
+function sleep(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
```
Comment (nlarew) [inline packages/mongodb-chatbot-evaluation/src/generate/generateLlmConversationData.ts:126]: Also have this in `generateConversationData.ts` - worth moving to a utils file?

**Review by nlarew — APPROVED** [review]
LGTM + non-blocking suggestion

File: packages/mongodb-chatbot-evaluation/src/generate/generateLlmConversationData.test.ts (hunk 1 above)
Comment (mongodben) [reply packages/mongodb-chatbot-evaluation/src/generate/generateLlmConversationData.test.ts:65]: no issue, just validation of functionality.

**Pull request merged**